include/clients/nrt/BufStatsClient.hpp

```diff
@@ -43,6 +44,7 @@ constexpr auto BufStatsParams = defineParameters(
     LongParam("startChan", "Start Channel", 0, Min(0)),
     LongParam("numChans", "Number of Channels", -1),
     BufferParam("stats", "Stats Buffer"),
+    ChoicesParam("select","Selection of Statistics","mean","std","skew","kurtosis","low","mid","high"),
```

Comment:
SelectParam?

Reply:
Maybe the "Param" bit is implied and therefore added noise? I might also suggest that it's not a Parameter, it's a statistic?

Reply:
I think that Param is there in the C++ because its a C++ thing, not a user-facing thing to keep it consistent with the other things which are defined as parameters/attributes.

Reply:
I have deleted the whole build folder. I'll redo that.

Reply:
There's no need for renaming the C++ type, I would argue

Reply:
Everything in that layer is contingent anyway